model/repository.go

```diff
@@ -42,6 +42,7 @@ var (
 
 type Revision struct {
 	Author          string
+	AuthorID        string
```

Comment:
what's the difference between this and Author?

Reply:
Author is the name "John Liu" while AuthorID is the Evergreen ID "john.liu"